src/ansible_navigator/utils.py

```diff
@@ -352,12 +352,12 @@ def human_time(seconds: int) -> str:
     hours, seconds = divmod(seconds, 3600)
     minutes, seconds = divmod(seconds, 60)
     if days > 0:
-        return "%s%dd%dh%dm%ds" % (sign_string, days, hours, minutes, seconds)
+        return f"{sign_string}{days}d{hours}h{minutes}m{seconds}s"
```

Comment:
This would be equivalent to the original:
```suggestion
        return f"{sign_string!s}{days:d}d{hours:d}h{minutes:d}m{seconds:d}s"
```

Reply:
@webknjaz sign_string is already a string, and I don't see a need to use :d here since an f string will str() the value

Reply:
Then, the behavior will not be equivalent. As I said above, explicit markers help with readability. But not only that. When you instruct it to call `int()` (which is what `:d` does essentially) on the rendered object, it's also a sanity check/safeguard for free, it will fail loudly if that thing will not be convertable to a number. Some time in the future, when you refactor things, you'd appreciate this rather than having a silent conversion and hearing about a bug from the end-users when it's shipped.

Reply:
again, happy to make this change. IMO it feels unnecessary, but if it helps you and contributors in the future it's worthwhile.

Reply:
@webknjaz Looking at this again, I don't think the use of `:d` if functionally equivalent to the initial string.   In the case divmod returns a float, the use of `%d` or the use of an f-string will convert the float to a string, where `:d` does not.

```
>>> "%dd" % 5
'5d'
>>> "%dd" % 5.0
'5d'
>>> foo = 5
>>> f"{foo:d}"
'5'
>>> foo = 5.0
>>> f"{foo:d}"
>>> f"{foo}"
'5.0'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Unknown format code 'd' for object of type 'float'
``` 

If we want to use the `:d` within the fstring, I think it is best to ensure the values returned by a divmod are explicitly integers.  Although it might be questionably out of scope for the PR, I think it is worthwhile because the type hints for the helper function show a float or int can be passed but we will always return a tuple of ints.

Reply:
The helper function should eliminate the possibility of someone modifying the function and passing a float to divmod which would result in a response from divmod of `float, float`, and cause the f-string to choke.

Reply:
I guess. My point is that we need to make sure that the f-string will choke as early as possible and part of that is using explicit specifiers. Another thing is the readability aspect of this: if a person looks at just one line, it's obvious what sort of data is being rendered w/o having to backtrace what the variable is, where it's defined and how it's passed or mutated.

Reply:
@webknjaz ok to gate and merge?

Reply:
Okay, let's do it.